src/components/styles/Icons/index.stories.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import { jsx, Text } from 'theme-ui';
+import { Meta } from '@storybook/react/types-6-0';
+
+import { baseIcons } from './consts';
+import { IconsDemoProps } from './types';
+
+const iconsStoriesMeta: Meta = {
+  title: 'Styles/Icons',
+  argTypes: {
+    items: {
+      table: { disable: true },
+      defaultValue: baseIcons,
+    },
+  },
+};
+
+export default iconsStoriesMeta;
+
+export const Base: React.FC<IconsDemoProps> = ({ items }) => (
+  <div
```

Comment:
Maybe you can use some components from `ThemeUI` here, like `Flex`, etc.

Reply:
Done.